README.md

```diff
@@ -1,2 +1,24 @@
 # zenpaperwallet.com
 JavaScript Client-Side ZEN Wallet Generator using [zencashjs library](https://github.com/HorizenOfficial/zencashjs) and inspired by [bitaddress.org](https://github.com/pointbiz/bitaddress.org)
+
+Download the source code and run:
+
+## Install node packages:
+```npm i```
```

Comment:
Instead of "npm i" you should use "npm ci" just to be sure that the exact version of the dependecies in the package-lock.json file will be used.